tests/vmi_multus_test.go

```diff
@@ -787,12 +787,13 @@ var _ = Describe("[Serial]SRIOV", func() {
 		})
 
 		It("[test_id:3985]should create a virtual machine with sriov interface with custom MAC address", func() {
-			vmi := getSriovVmi([]string{"sriov"})
 			mac := "de:ad:00:00:be:ef"
+			vmi := getSriovVmi([]string{"sriov"})
 			vmi.Spec.Domain.Devices.Interfaces[1].MacAddress = mac
 			startVmi(vmi)
 			waitVmi(vmi)
-	
+			err, interfaceName := getSRIOVInterfaceNameByMAC(virtClient, vmi, mac)
+			Expect(err).NotTo(HaveOccurred(), "should get sriov interface name")
```

Comment:
nit: Please separate by newlines this section.

Reply:
Done.